enterprise/frontend/src/embedding-sdk/components/public/index.ts

```diff
@@ -18,6 +18,8 @@ export type {
   CreateDashboardModalProps,
   CreateDashboardValues,
 } from "./CreateDashboardModal";
+export { FlexibleSizeComponent } from "./FlexibleSizeComponent";
```

Comment:
Is this in the scope of this PR? I assume this means we're exporting `FlexibleSizeComponent` and users can import this from our SDK. I don't know why we want to do that, and I couldn't find where we use that in this PR.